lib/ansible/modules/network/icx/icx_banner.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+
+DOCUMENTATION = """
+---
+module: icx_banner
+version_added: "2.9"
+author: "Ruckus Wireless (@Commscope)"
+short_description: Manage multiline banners on Ruckus ICX 7000 series switches
+description:
+  - This will configure both login and motd banners on remote
+    ruckus ICX 7000 series switches. It allows playbooks to add or remove
+    banner text from the active running configuration.
+options:
+  banner:
+    description:
+      - Specifies which banner should be configured on the remote device.
+    type: str
+    required: true
+    choices: ['motd', 'exec', 'incoming']
+  text:
+    description:
+      - The banner text that should be
+        present in the remote device running configuration.
+        This argument accepts a multiline string, with no empty lines.
+    type: str
+  state:
+    description:
+      - Specifies whether or not the configuration is
+        present in the current devices active running configuration.
+    type: str
+    default: present
+    choices: ['present', 'absent']
+  enterkey:
+    description:
+      - Specifies whether or not the motd configuration should accept
+        the require-enter-key
+    type: bool
+    default: no
+  check_running_config:
```

Comment:
What is the purpose of this option? Also if you want option value to fall back on environment variable it can be defined  in the argspec as 
```
'check_running_config': dict(default=True, type='bool', fallback=(env_fallback, ['ANSIBLE_CHECK_RUNNING_CONFIG']))
```

Reply:
User should be able to specify if he/she wants ICX modules to send commands, after checking the running config on switch or without checking. User should be able to specify this at a global level and be able to override it at module level. Basically it is similar to check_mode. 

Will the change that you have recommended achieve this?

Can you please tell me how to get myself invited to get added to ansible-network.slack channel

Reply:
So basically if this option is enabled it will not compare the running-config with candidate config but will directly push the commands to the remote host. If that's the case I don't see a reason for it to be an environment variable.

Can you please share me your mail-id. You can unicast it to me on IRC. My IRC nick is @ganeshrn

Reply:
You use the link here https://github.com/ansible/community/wiki/Network to join the ansiblenetwork slack channel

Reply:
thanks for the links. About check_running_config, if not environment variable how else can we achieve the same?

Reply:
You can define the option as
```
'check_running_config': dict(default=True, type='bool', fallback=(env_fallback, ['ANSIBLE_CHECK_RUNNING_CONFIG']))
```
This will read set the option value if `ANSIBLE_CHECK_RUNNING_CONFIG` is set. Also you don't need to implement `get_env_diff` in cliconf plugin as the value of `compare` passed to `get_config` api in cliconf already handles checking the value of enviornment variable.